proseco/tests/test_catalog.py

```diff
@@ -189,7 +189,7 @@ def test_pickle():
     """
     stars = StarsTable.empty()
     stars.add_fake_constellation(mag=10.0, n_stars=5)
-    aca = get_aca_catalog(stars=stars, raise_exc=True, **STD_INFO)
+    aca = get_aca_catalog(stars=stars, dark=DARK40, raise_exc=True, **STD_INFO)
```

Comment:
This was always only selecting 4 stars and just squeaking by with guide count of 4.0.  That was not the intent of this test

Reply:
Gotcha.  And yes, those 10.0 mag stars can be tricky on an average dark map.  Looks like a reasonable fix.

Reply:
For the record can you punch the approve?